main.go

```diff
@@ -83,9 +84,12 @@ func (t transceiverCollectorWrapper) Describe(ch chan<- *prometheus.Desc) {
 func handleMetricsRequest(w http.ResponseWriter, request *http.Request) {
 	registry := prometheus.NewRegistry()
 
-	excludedIfaceNames := strings.Split(*excludeInterfaces, ",")
-	for index, excludedIfaceName := range excludedIfaceNames {
-		excludedIfaceNames[index] = strings.Trim(excludedIfaceName, " ")
+	var excludedIfaceNames []string
+	if len(*excludeInterfaces) != 0 {
```

Comment:
Since we got new features, it would also be nice to support regex for included interfaces.